health/main.py

```diff
@@ -121,17 +119,19 @@ def job():
                      % (time.time() - started_at))
 
 
-def main():
+def main(argv=sys.argv[1:]):
```

Comment:
1) Please do not use any argument that is important. 
2) Do not use mutable default values for arguments. This is python anitpattern http://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html 

So just remove it and put as first string 

  argv = sys.argv[1:]

That will also simplify testing this method (because mocking will become way simpler)